tests/client/src/components/QA/QA.test.js

```diff
@@ -69,15 +69,7 @@ describe('<QA testing />', () => {
   test('should add 2 questions when click button', () => {
     const wrapper = mount(<QuestionList questions={questionsData} />);
     expect(wrapper.find('table.question_table')).toHaveLength(2);
-    wrapper.find('button.moreQuestions').simulate('click');
+    wrapper.find('button.morequestions').simulate('click');
     expect(wrapper.find('table.question_table')).toHaveLength(4);
   });
```

Comment:
https://github.com/rpp29-fec-gouda/atelier/blob/700c6f822d00f27befbd36f104f95be4a2ba65ec/tests/client/src/components/QA/QA.test.js#L13

Good use of unit test data. Move it out to a different file ( I suggest a folder called fixtures for all your teammates) to have the test data